xanadu_sphinx_theme/globaltoc.html

```diff
@@ -1,6 +1,19 @@
 
 <div class="sidebar-block">
   <div class="sidebar-toc">
-    {{ toctree(maxdepth=1, collapse=True, includehidden=True) }}
+    {% if toc_overview %}
+    <p class="caption">{{ theme_project_nav_name or shorttitle }}</p>
+    <ul>
+      <li class="caption toctree-l1 {{'current' if pagename == 'index' }}">
+        <a class="reference internal" href="{{ pathto('index') }}">Overview</a>
+      </li>
+    </ul>
+    {% endif %}
+    {% set toctree = toctree(maxdepth=1, collapse=True, includehidden=True) %}
+    {% if toctree %}
+      {{ toctree }}
+    {% else %}
+      {{ toc }}
+    {% endif %}
```

Comment:
Just curious, what is the issue with using
```suggestion
    {% toctree = toctree(maxdepth=1, collapse=True, includehidden=True) %}
```

Reply:
I wasn't sure, I didn't know if there was a reason that past me had done this 😆